dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -171,17 +174,20 @@ class DummyOutcomeRefuter(CausalRefuter):
     # The Default split for the number of data points that fall into the training and validation sets
     DEFAULT_TEST_FRACTION = [TestFraction(0.5, 0.5)]
 
+    DEFAULT_NEW_DATA_WITH_UNOBSERVED_CONFOUNDING = pd.Series()
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-
+        
         self._num_simulations = kwargs.pop("num_simulations", CausalRefuter.DEFAULT_NUM_SIMULATIONS)
         self._transformation_list = kwargs.pop("transformation_list", DummyOutcomeRefuter.DEFAULT_TRANSFORMATION)
         self._true_causal_effect = kwargs.pop("true_causal_effect", DummyOutcomeRefuter.DEFAULT_TRUE_CAUSAL_EFFECT)
         self._bucket_size_scale_factor = kwargs.pop("bucket_size_scale_factor", DummyOutcomeRefuter.DEFAULT_BUCKET_SCALE_FACTOR)
         self._min_data_point_threshold = kwargs.pop("min_data_point_threshold", DummyOutcomeRefuter.MIN_DATA_POINT_THRESHOLD)
         self._test_fraction = kwargs.pop("_test_fraction", DummyOutcomeRefuter.DEFAULT_TEST_FRACTION)
+        self._new_data_with_unobserved_confounding = kwargs.pop("new_data_with_unobserved_confounding", DummyOutcomeRefuter.DEFAULT_NEW_DATA_WITH_UNOBSERVED_CONFOUNDING)
```

Comment:
a simpler parameter name is `unobserved_confounder_values` which defaults to None.